src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  width: $card-width;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+
+  &__top {
+    padding: 32px 20px 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__bottom {
+    padding: 0 16px 16px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__code {
+    margin-top: 4px;
+  }
+
+  &__rating {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__expense {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__button {
+    margin-top: 16px;
+
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $button-ltblue-color;
+    color: $white;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: $white;
+      color: $button-ltblue-color;
+      border: 1px solid $button-ltblue-color;
+    }
+  }
+
+  &__text {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 400;
+
+    &--title {
```

Comment:
every modificator here is used once and don't use almost any of the "parent" styles:
- font-family should be set for the page (it'll be inherited)
- line-height you anyway set its own almost for every modificator
- rest properties have default values

so let's make each of these modificators just a common element of `card` BEM-block

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/b9ccf001-c208-479b-893a-6b4d8cad7eee)

still not resolved

you have just elements, no need to make them modifiers